### PR TITLE
docs: add clash of space idea and project

### PR DESCRIPTION
**Pull request by Rajspeaks**
Clash of Space game both Idea and Project added

### CODE REVIEW

File: data/ideas.json [hunk 1]
```diff
@@ -185,4 +185,10 @@
         "level": "advanced",
         "tags": "roadmap, success, process"
     }
+    {
+        "title": "Clash of Space",
+        "description": "This is space-craft and alien shooting game where a player can earn points based on how many alien spacecrafts the player can destroy and the player has to protect his/her spaceship from the aliens to save the health level.",
+        "level": "basic",
+        "tags": "clas-of-space, game, spacewar, alien-shooting, action"
```
Comment (ming-tsai) [inline data/ideas.json:192]: Please separate the word using the dash(`-`)
```suggestion
        "tags": "clash-of-space, game, space-war, alien-shooting, action"
```

File: data/projects.json [hunk 2]
```diff
@@ -289,5 +289,15 @@
...
     }
+   {
+        "name": "clash-of-space",
+        "display": "Clash of Space",
+        "description": "This is a space-craft war and alien shooting game",
+        "author": "Rajspeaks",
+        "category": "game",
+        "level": "basic",
+        "site": "https://rajspeaks.github.io/clash-of-space/",
+        "tags": "game, html, css, javascript, space"
```
Comment (ming-tsai) [inline data/projects.json:300]: To a match the project to a idea please add the first tag of the idea, in this case
```suggestion
        "tags": "game, html, css, javascript, space, clash-of-space"
```

File: data/ideas.json [hunk 3]
```diff
@@ -185,4 +185,10 @@
         "level": "advanced",
         "tags": "roadmap, success, process"
     }
```
Comment (ming-tsai) [inline data/ideas.json:187]: Please add an missing `,`

File: data/projects.json [hunk 4]
```diff
@@ -289,5 +289,15 @@
         "site": "https://ecommerce-multi-page.vercel.app/index.html",
         "tags": "e-commerce, html, css, vercel"
     }
```
Comment (ming-tsai) [inline data/projects.json:291]: Please add an missing `,`

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @Rajspeaks,
Some suggestions for the pull request.

**Review by Rajspeaks — COMMENTED** [review]
Changes done

File: data/ideas.json (hunk 3 above)
Comment (Rajspeaks) [reply data/ideas.json]: I have added the requirements

File: data/projects.json (hunk 4 above)
Comment (Rajspeaks) [reply data/projects.json]: Added

**Review by ming-tsai — APPROVED** [review]

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**